Source/WebCore/platform/text/TextFlags.h

```diff
@@ -163,7 +166,64 @@ enum class FontVariantNumericFraction : uint8_t {
 
 enum class FontVariantNumericOrdinal : bool { Normal, Yes };
 enum class FontVariantNumericSlashedZero : bool { Normal, Yes };
-enum class FontVariantAlternates : bool { Normal, HistoricalForms };
+
+struct FontVariantAlternatesNormal {
+    bool operator==(const FontVariantAlternatesNormal&) const = default;
+};
+
+struct FontVariantAlternatesValues {
+    bool operator==(const FontVariantAlternatesValues&) const = default;
+    std::optional<String> stylistic;
+    std::optional<String> styleset;
+    std::optional<String> characterVariant;
+    std::optional<String> swash;
+    std::optional<String> ornaments;
+    std::optional<String> annotation;
```

Comment:
Can these be enums rather than strings?

Reply:
I'm not sure I understand this comment 🤔  Those value can be arbitrary strings.

Reply:
Ah, you're right, ok.